src/UserInfo/UserInfo.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { User } from '../types';
+import './UserInfo.scss';
+
+type Props = {
+  user: User | null;
+};
+
+const UserInfo:React.FC<Props> = ({ user }) => {
+  return (
+    <>
+      <div className="userInfo">
+        <div>{user?.name}</div>
+        <div>{user?.email}</div>
+      </div>
+
+    </>
+  );
+};
```

Comment:
```suggestion
const UserInfo:React.FC<Props> = ({ user }) => (
      <div className="userInfo">
        <div>{user?.name}</div>
        <div>{user?.email}</div>
      </div>
  );
```
no need in fragment and you can remove return statement here